package.json

```diff
@@ -76,7 +76,8 @@
   },
   "pnpm": {
     "overrides": {
-      "string-width": "4.2.3"
+      "string-width": "4.2.3",
+      "vue": "$vue"
```

Comment:
This line is the fix to ensure all packages resolve to the same vue version (only locally in this repository) during local development